src/components/sprite-selector/sprite-list.jsx

```diff
@@ -8,9 +8,12 @@ import Box from '../box/box.jsx';
 import SpriteSelectorItem from '../../containers/sprite-selector-item.jsx';
 import SortableHOC from '../../lib/sortable-hoc.jsx';
 import SortableAsset from '../asset-panel/sortable-asset.jsx';
+import ThrottledPropertyHOC from '../../lib/throttled-property-hoc.jsx';
 
 import styles from './sprite-selector.css';
 
+const ThrottledSpriteSelectorItem = ThrottledPropertyHOC('asset', 500, SpriteSelectorItem);
```

Comment:
This doesn't really matter but I think the convention for configuring HOCs is `ThrottledPropertyHOC('asset', 500)(SpriteSelectorItem)`

Reply:
Right! Fixed